common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -2147,7 +2147,6 @@ schematic = (function() {
 	    this.bg_image.height = this.height;
 
 	    if (!this.diagram_only) {
-		this.canvas.tabIndex = 1; // so we get keystrokes
```

Comment:
I'm not sure I fully understand the impact of this change. Does it need to be -1 or 0 instead of being removed altogether? Are the interactive elements still focusable?